utils/m0reportbug

```diff
@@ -191,7 +197,12 @@ main() {
     fi
 
     # Collects traces of m0d process(confd, ios) only
-    M0TR_M0D_TRACE_DIR=$(cat /etc/sysconfig/motr  | grep "^MOTR_M0D_TRACE_DIR" | cut -d '=' -f2)
+    MACHINE_ID=$(cat /etc/machine-id)
+    if [ -d  /etc/cortx/motr/sysconfig/$MACHINE_ID ]; then
+        M0TR_M0D_TRACE_DIR=$(cat /etc/cortx/motr/sysconfig/$MACHINE_ID/motr | grep "^MOTR_M0D_TRACE_DIR" | cut -d '=' -f2)
+    else
+        M0TR_M0D_TRACE_DIR=$(cat /etc/sysconfig/motr  | grep "^MOTR_M0D_TRACE_DIR" | cut -d '=' -f2)
```

Comment:
Get M0TR_M0D_TRACE_DIR, MOTR_M0D_ADDB_STOB_DIR,  as same as M0TR_M0D_CORE_DIR

Reply:
these are existing code that  why did not change

Reply:
incorporated